.dependencygraph/setting.json

```diff
@@ -0,0 +1,13 @@
+{
+  "alias": false,
+  "resolveExtensions": [
+    ".js",
+    ".jsx",
+    ".ts",
+    ".tsx",
+    ".vue",
+    ".scss",
+    ".less"
+  ],
+  "entryFilePath": "index.js"
+}
```

Comment:
What is this file for?

Reply:
I think its an extension for CSS, I would guess that it had something to with the charts. But we didn't add that for this ticket.